src/io/flutter/samples/FlutterSample.java

```diff
@@ -17,22 +17,31 @@
   @NotNull
   private final String file;
   @NotNull
+  private final String sourcePath;
+  @NotNull
   private final String description;
 
-  private String relativePath;
-
   FlutterSample(@NotNull String element, @NotNull String library, @NotNull String id, @NotNull String file,
-                @NotNull String description) {
+                @NotNull String sourcePath, @NotNull String description) {
     this.element = element;
     this.library = library;
     this.id = id;
     this.file = file;
+    this.sourcePath = sourcePath;
     this.description = description;
   }
 
   @Override
   public String toString() {
-    return getElement() + " (" + getLibrary() + ")";
+    return getDisplayLabel();
+  }
+
+  /**
+   * Get a label suitable for display in a chooser (e.g., combobox).
+   */
+  public String getDisplayLabel() {
+    // TODO(pq): add disambiguation once it's needed.
```

Comment:
also looks like the existing labels aren't that user friendly.

Reply:
Agreed.  Context from the description helps but yeah, these aren't great.  VERY open to ideas, though we'll likely want to stay consistent w/ what we're doing for VSCode.

/fyi @DanTup